init/bin.js

```diff
@@ -36,10 +37,12 @@ const getVariableFlag = function ({
   description,
   default: defaultValue,
 }) {
+  const defaultDescription =
+    typeof defaultValue === 'string' ? `\nDefault: ${defaultValue}` : ''
   return {
     [name]: {
       string: true,
-      describe: `${description}.\nDefault: ${defaultValue}`,
+      describe: `${description}.${defaultDescription}`,
```

Comment:
This was a bug: some `defaultValue` as function to retrieve the default value, as opposed to strings.